packaging/runtime-check.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+wait_for() {
```

Comment:
maybe a combination of an exponential backoff or and retry logic?

Reply:
I know that more than 30 sec for a startup is bad, but with runners we can't be certain.

Reply:
No, we can be even in this case. The way they run doesn’t result in long random delays, and 30 seconds is sufficient headroom for expected short delays. Keep in mind that this is a clean install with no data to load, so the startup time here is just however long it takes Netdata to get the internal data structures and the thread/process tree set up, which should realistically never be more than a single second on decent hardware.

Reply:
Also, to clarify a bit, a simple flat timeout was chosen here to intentionally make the script simpler (and thus less likely to have errors) and ensure that failures still register relatively quickly when they occur.